<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraffitiWidget</class>
 <widget class="QWidget" name="GraffitiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="WidgetPencil" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>185</y>
     <width>208</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#WidgetPencil
{
background-color: rgb(222, 222, 222);
}</string>
   </property>
   <widget class="QPushButton" name="PButtonHide">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(149, 149, 149);
background: transparent;
border:none;
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>166</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color: rgb(133, 133, 133);
}</string>
    </property>
    <property name="text">
     <string>铅笔颜色：默认选择黑色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255, 0, 0);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_2">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>50</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(170, 0, 255);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255, 255, 0);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>50</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255, 0, 255);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 255, 0);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255, 170, 255);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_7">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>90</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255, 85, 0);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(85, 170, 255);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_9">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>90</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 0, 255);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(200,200, 200);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 85, 0);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_12">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>130</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(170, 0, 85);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_13">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(120,120, 120);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_14">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>130</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 0, 0);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonColor_15">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255, 255, 255);
border:none;
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color: rgb(133, 133, 133);
}</string>
    </property>
    <property name="text">
     <string>线条粗细：默认选择细</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonBlod_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>205</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:5px;
background: transparent;
color: rgb(133, 133, 133);
border:1px solid rgb(133,133,133);
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string>细</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonBlod_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>205</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:5px;
background: transparent;
color: rgb(133, 133, 133);
border:1px solid rgb(133,133,133);
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string>中粗</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonBlod_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>205</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:5px;
background: transparent;
color: rgb(133, 133, 133);
border:1px solid rgb(133,133,133);
}
QPushButton::checked
{
border:1px solid rgb(178,255,255);
}</string>
    </property>
    <property name="text">
     <string>粗</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PButtonSure">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>245</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:5px;
background-color: rgb(18, 156, 221);
color: rgb(222, 222, 222);
}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
